src/index.js

```diff
@@ -184,6 +184,56 @@ VirtualMachine.prototype.addBackdrop = function (backdropObject) {
 };
 
 /**
+ * Rename a sprite.
+ * @param {string} targetId ID of a target whose sprite to rename.
+ * @param {string} newName New name of the sprite.
+ */
+VirtualMachine.prototype.renameSprite = function (targetId, newName) {
+    var target = this.runtime.getTargetById(targetId);
+    if (target) {
+        var sprite = target.sprite;
+        if (!sprite) {
+            throw new Error('No sprite associated with this target.');
+        }
+        sprite.name = newName;
+        this.emitTargetsUpdate();
+    } else {
+        throw new Error('No target with the provided id.');
+    }
+};
+
+/**
+ * Delete a sprite and all its clones.
+ * @param {string} targetId ID of a target whose sprite to delete.
+ */
+VirtualMachine.prototype.deleteSprite = function (targetId) {
+    var target = this.runtime.getTargetById(targetId);
+    if (target) {
+        if (target.isStage) {
+            throw new Error('Cannot delete stage targets.');
+        }
```

Comment:
We should probably carry this check to the rename method as well, unless you intended to make renaming the stage possible?  If you do carry it over, maybe both of these checks should be put onto `RenderedTarget` as a method like `isSprite`? Or something less confusing than that.